contrib/docker-engine-selinux/docker.fc

```diff
@@ -17,4 +17,5 @@
 /var/lib/docker/init(/.*)?		gen_context(system_u:object_r:docker_share_t,s0)
 /var/lib/docker/containers/.*/hosts		gen_context(system_u:object_r:docker_share_t,s0)
 /var/lib/docker/containers/.*/hostname		gen_context(system_u:object_r:docker_share_t,s0)
+/var/lib/docker/containers/.*/.*\.log	--	gen_context(system_u:object_r:docker_log_t,s0)
```

Comment:
should we also change this in docker-selinux upstream https://github.com/projectatomic/docker-selinux?